src/theme/index.tsx

```diff
@@ -110,6 +111,59 @@ function colors(darkMode: boolean): Colors {
     // dont wanna forget these blue yet
     deprecated_blue4: darkMode ? '#153d6f70' : '#C4D9F8',
     // blue5: darkMode ? '#153d6f70' : '#EBF4FF',
+
+    blue200: ColorsPalette.blue200,
+
+    /* UPDATED */
```

Comment:
what is this comment referring to?

Reply:
oh it's just the Updated colors haha, I can take out since the other colors have deprecated now